pkg/render/manager.go

```diff
@@ -730,180 +730,7 @@ func managerClusterRole(managedCluster bool, kubernetesProvider operatorv1.Provi
-			},
-			{
-				APIGroups: []string{"projectcalico.org"},
-				Resources: []string{
-					"hostendpoints",
-				},
-				Verbs: []string{"list"},
-			},
-			{
-				APIGroups: []string{"projectcalico.org"},
-				Resources: []string{
-					"felixconfigurations",
-				},
-				ResourceNames: []string{
-					"default",
-				},
-				Verbs: []string{"get"},
-			},
-			{
-				APIGroups: []string{"projectcalico.org"},
-				Resources: []string{
-					"alertexceptions",
-				},
-				Verbs: []string{"get", "list", "update"},
-			},
-			{
-				APIGroups: []string{"networking.k8s.io"},
-				Resources: []string{"networkpolicies"},
-				Verbs:     []string{"get", "list"},
-			},
-			{
-				APIGroups: []string{"policy.networking.k8s.io"},
-				Resources: []string{
-					"adminnetworkpolicies",
-					"baselineadminnetworkpolicies",
-				},
-				Verbs: []string{"list"},
-			},
-			{
-				APIGroups: []string{""},
-				Resources: []string{"serviceaccounts", "namespaces", "nodes", "events", "services", "pods"},
-				Verbs:     []string{"list"},
-			},
-			{
-				APIGroups: []string{"apps"},
-				Resources: []string{"replicasets", "statefulsets", "daemonsets"},
-				Verbs:     []string{"list"},
-			},
-			// When a request is made in the manager UI, they are proxied through the Voltron backend server. If the
-			// request is targeting a k8s api or when it is targeting a managed cluster, Voltron will authenticate the
-			// user based on the auth header and then impersonate the user.
-			{
-				APIGroups: []string{""},
-				Resources: []string{"users", "groups", "serviceaccounts"},
-				Verbs:     []string{"impersonate"},
-			},
-			// Allow query server talk to Prometheus via the manager user.
-			{
-				APIGroups: []string{""},
-				Resources: []string{"services/proxy"},
-				ResourceNames: []string{
-					"https:tigera-api:8080", "calico-node-prometheus:9090",
-				},
-				Verbs: []string{"get", "create"},
-			},
-			{
-				// Add access to Linseed APIs.
-				APIGroups: []string{"linseed.tigera.io"},
-				Resources: []string{
-					"flows",
-					"flowlogs",
-					"bgplogs",
-					"auditlogs",
-					"dnsflows",
-					"dnslogs",
-					"l7flows",
-					"l7logs",
-					"events",
-					"processes",
-				},
-				Verbs: []string{"get"},
-			},
-			{
-				// Dismiss events.
-				APIGroups: []string{"linseed.tigera.io"},
-				Resources: []string{
-					"events",
-				},
-				Verbs: []string{"dismiss", "delete"},
-			},
-		},
-	}
-
-	if !managedCluster {
-		cr.Rules = append(cr.Rules,
-			rbacv1.PolicyRule{
-				APIGroups: []string{"projectcalico.org"},
-				Resources: []string{"managedclusters"},
-				Verbs:     []string{"list", "get", "watch", "update"},
-			},
-		)
-	}
-
-	if tenant.MultiTenant() {
-		cr.Rules = append(cr.Rules,
-			rbacv1.PolicyRule{
-				APIGroups: []string{"authorization.k8s.io"},
-				Resources: []string{"localsubjectaccessreviews"},
-				Verbs:     []string{"create"},
-			},
-		)
-
-		if tenant.ManagedClusterIsCalico() {
```

Comment:
@caseydavenport Since the Free tier now includes Guardian, will adding these rules to the Guardian RBAC also apply to the Free tier? For certain components, do we need to add a check for !tenant.ManagedClusterIsCalico() when merging these RBAC rules?